client/src/common/colors.js

```diff
@@ -0,0 +1 @@
+export const PRIMARY = "#6A7EFC";
```

Comment:
hmm how come this is in `/src/common`? I thought this should be `constants/colors.js` ... folder was renamed like 2 weeks ago or so?

Reply:
i had to pull and rebase, my locals were not up to date